ui/components/app/edit-gas-display/edit-gas-display.component.js

```diff
@@ -77,7 +77,10 @@ export default function EditGasDisplay({
   );
 
   const [showAdvancedForm, setShowAdvancedForm] = useState(
-    !estimateToUse || hasGasErrors || !networkAndAccountSupport1559,
+    !estimateToUse ||
+      estimateToUse === 'custom' ||
+      hasGasErrors ||
```

Comment:
I think we can remove `hasGasErrors` here, due to my recent PR, but not a blocker.